pkg/commands/git.go

```diff
@@ -208,23 +213,29 @@ func (c *GitCommand) ResetHard() error {
 // current branch
 func (c *GitCommand) UpstreamDifferenceCount() (string, string) {
 	pushableCount, err := c.OSCommand.RunCommandWithOutput("git rev-list @{u}..head --count")
+
```

Comment:
Relating to your earlier comment @antham, I don't think there should be a blank line here because there is a direct connection between the code above/below this blank line. I'm not fussed about the following blank line though.